toolkit/tools/pkg/imagecustomizerlib/customizeutils.go

```diff
@@ -46,9 +62,53 @@ func doCustomizations(baseConfigPath string, config *imagecustomizerapi.SystemCo
 		return err
 	}
 
+	err = deleteResolvConf(imageChroot)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// Override the resolv.conf file, so that in-chroot processes can access the network.
+// For example, to install packages from packages.microsoft.com.
+func overrideResolvConf(imageChroot *safechroot.Chroot) error {
```

Comment:
We have generally used `SafeChroot.AddFiles()` to add the network configs, but this works fine as well. (see RpmRepoCloner.addNetworkFiles()`